File: Front/src/app/core/models/SampleDTO.ts
import {ExtractorType} from './enums/ExtractorType';

export interface ISampleDTO {
    createdAt: string;
    updatedAt: string;
    extractorType: ExtractorType;
    originalFileName: string;
    label: string;
    attributes: any;
    fileId: string;
    getAttributes();
}

export class SampleListDTO  implements  ISampleDTO {
    id: string;
    attributes: number;
    createdAt: string;
    extractorType: ExtractorType;
    label: string;
    originalFileName: string;
    updatedAt: string;
    fileId: string;

    getAttributes() {
    }


}

export class SampleRegisterDTO  implements  ISampleDTO {
    attributes: any[] = [];
    createdAt: string;
    extractorType: ExtractorType = ExtractorType.GLCM;
    label: string;
    originalFileName: string;
    updatedAt: string;
    fileId: string;

    getAttributes() {
    }


}

File: commons.module/src/main/java/br/com/lukkascost/commons/module/models/dto/SampleDTO.java
package br.com.lukkascost.commons.module.models.dto;

import br.com.lukkascost.commons.module.models.enuns.ExtractorType;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Getter
@Setter
public class SampleDTO {
    private UUID id;
    private String createdAt;
    private String updatedAt;
    private ExtractorType extractorType;
    private long attributes;
    private String originalFileName;
    private String label;
}
